gcsio/pom.xml

```diff
@@ -156,6 +156,13 @@
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud.bigdataoss</groupId>
+      <artifactId>util</artifactId>
+      <version>${bigdataoss.version}</version>
+      <classifier>tests</classifier>
+      <scope>test</scope>
+    </dependency>
```

Comment:
do we need this ? considering it is already added L140 with default scope

Reply:
Yes, this is dependency for `tests` classifier because they have testing classes.